prisondersdilemma.py

```diff
@@ -0,0 +1,202 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May  7 13:37:28 2020
+
+@author: katha
+"""
+
+from psychopy.visual import Window, TextStim, Circle, ImageStim, Rect, ShapeStim
+from psychopy.core import wait, Clock
+from psychopy.event import getKeys, clearEvents, waitKeys, Mouse
+from numpy.random import shuffle
+import numpy as np
+import random
+
+#these are all just building blocks for now, not functioning code! it won't run yet
+
+#create window
+
+DISPSIZE = (1000, 700)
```

Comment:
I was adivced I could adjust the display size to fit to other people's displays, so maybe that can be useful for you as well! Here is how it is supposed to happen: from win32api import GetSystemMetrics --> DISPSIZE = (GetSystemMetrics(0),GetSystemMetrics(1)).